### ISSUE TITLE
How to start activity without animation

### ISSUE THREAD

**Issue by sorcererXW** [VictorAlbertos/RxActivityResult#24]
The process of shifting between A and B is no animation
But how to remove animation after using RxActivityResult 
```java
RxActivityResult.on(mActivity)
        .startIntent(new Intent(mActivity, SearchActivity.class)
                .addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION)
                .putExtra("custom picker", mainActivity.isCustomPicker()))
        .subscribe(new Action1<Result<Activity>>() {
            @Override
            public void call(Result<Activity> result) {
                if (result.data() != null) {
                    mainActivity.onReturnCustomPickerRes(
                            result.data().getIntExtra("res", 0));
                }
            }
        });
```

**Comment by VictorAlbertos** [comment]
I don't use custom transitions between activities, so I'm not familiar with that API. Sorry.

**Comment by VictorAlbertos** [comment]
Closed via #25
